src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] ball = new Ball[3];
+        Lottery llottery = new Lottery();
+        for(int i = 0; i < ball.length; i++){
+            ball[i] = llottery.getRandomBall();
+            System.out.println(ball[i].toString());
```

Comment:
```suggestion
            System.out.println(ball[i]);
```
no need to call toString explicitly